src/EVM/Sign.hs

```diff
@@ -0,0 +1,72 @@
+module EVM.Sign where
```

Comment:
A module description would be great here too

Reply:
Good point, added it as well, will push and resolve :)